packages/stateful/voting-module-adapter/adapters/CwdVotingCw721Staked/types.ts

```diff
@@ -0,0 +1,26 @@
+import { DurationWithUnits } from '@dao-dao/types'
+
+export enum GovernanceTokenType {
+  New,
+  Existing,
+}
+
+export interface DaoCreationConfig {
+  tokenType: GovernanceTokenType
+  newInfo?: {
+    initialSupply: number
+    initialTreasuryPercent: number
+    imageUrl?: string
+    symbol: string
+    name: string
+  }
+  existingGovernanceTokenAddress: string
```

Comment:
if we're not gonna support creating your own collection right now, we can get rid of any distinction between new and existing tokens.